editor/src/components/canvas/controls/select-mode/select-mode-hooks.ts

```diff
@@ -0,0 +1,407 @@
+import * as React from 'react'
+import { MetadataUtils } from '../../../../core/model/element-metadata-utils'
+import { uniqBy } from '../../../../core/shared/array-utils'
+import { JSXMetadata } from '../../../../core/shared/element-template'
+import {
+  boundingRectangleArray,
+  CanvasPoint,
+  distance,
+  point,
+  WindowPoint,
+  windowPoint,
+} from '../../../../core/shared/math-utils'
+import { ScenePath, TemplatePath } from '../../../../core/shared/project-file-types'
+import * as TP from '../../../../core/shared/template-path'
+import { NO_OP } from '../../../../core/shared/utils'
+import { KeysPressed } from '../../../../utils/keyboard'
+import { useKeepShallowReferenceEquality } from '../../../../utils/react-performance'
+import utils from '../../../../utils/utils'
+import {
+  clearHighlightedViews,
+  clearSelection,
+  selectComponents,
+  setHighlightedView,
+} from '../../../editor/actions/action-creators'
+import {
+  EditorState,
+  getOpenUtopiaJSXComponentsFromState,
+} from '../../../editor/store/editor-state'
+import { useEditorState, useRefEditorState } from '../../../editor/store/store-hook'
+import CanvasActions from '../../canvas-actions'
+import { DragState, moveDragState } from '../../canvas-types'
+import { createDuplicationNewUIDs, getOriginalCanvasFrames } from '../../canvas-utils'
+import { findFirstParentWithValidUID } from '../../dom-lookup'
+import { useWindowToCanvasCoordinates } from '../../dom-lookup-hooks'
+import { selectElementsThatRespectLayout } from '../new-canvas-controls'
+
+const DRAG_START_TRESHOLD = 2
+
+export function pickIsResizing(dragState: DragState | null): boolean {
```

Comment:
I understand the choice for the `pick` prefix here, but we're not using it anywhere else in the code base, so please can we remove it? Otherwise we're just going to end up with competing conventions.

Reply:
@Rheeseyb I renamed it to isResizing (and the consts in the code to `resizing` so they don't shadow each other)

I now remembered that the reason for pickIsResizing was to be able to call the consts isResizing. could you check if this new updated version is better?

Reply:
Works for me :+1: